packages/solicitudes/src/components/Sections/AsignaturasFormacionElectiva.jsx

```diff
@@ -0,0 +1,91 @@
+import { Grid } from '@mui/material';
+import { DataGrid } from '@mui/x-data-grid';
+import { ButtonAdd, Input } from '@siiges-ui/shared';
+import React from 'react';
+
+const columns = [
```

Comment:
Althought you are faking data I think you should put this in other file, also consider use fakerjs to fake the data